tests/e2e/kcs/test_noobaa_db_backup_and_recovery.py

```diff
@@ -76,3 +81,85 @@ def test_noobaa_db_backup_and_recovery(
         # Verify everything running fine
         log.info("Verifying All resources are Running and matches expected result")
         self.sanity_helpers.health_check(tries=120)
+
+    @pytest.fixture()
+    def warps3(self, request):
+
+        warps3 = warp.Warp()
+        warps3.create_resource_warp(replicas=4, multi_client=True)
+
+        def teardown():
+            warps3.cleanup()
+
+        request.addfinalizer(teardown)
+        return warps3
+
+    @pytest.mark.polarion_id("OCS-4842")
+    @pytest.mark.bugzilla("214035")
+    @skipif_ocs_version("<4.8")
+    def test_noobaa_db_backup_recovery_locally(
```

Comment:
not a must but we can remove this and simply not backport this PR to `release-4.8` branch

Reply:
ack. i will keep this as such for now :smiley: